.github/workflows/release.yaml

```diff
@@ -73,6 +77,10 @@ jobs:
             TELEMETRY_TOKEN=${{ secrets.TELEMETRY_TOKEN }}
             VERSION=${{ env.VERSION }}
 
+      - name: prepare-cli-manifests # this step will ensure that kusk kustomize manifests are build with the latest image tag github.sha
+        working-directory: cmd/kusk
+        run: VERSION=${{ env.VERSION }} make manifests
```

Comment:
Won't `run: VERSION=${{ github.sha }} make build` run `make manifests` because:

```Makefile
...
build: manifests
...
```

`build` requires `manifests`.

Reply:
@kylehodgetts can you chime in on this, please?